### ISSUE TITLE
release commit message subject is not compatible with commitlint conventional

### ISSUE THREAD

**Issue by xmlking** [semantic-release/semantic-release#928]
## Current behavior
looks line numbers in commit message generated (`chore(release): 1.0.1 [skip ci]`) with run `npm run semantic-release` is not acceptable with @commitlint/config-conventional .

how to test with `commitlint`

`printf "chore(release): 1.0.1 [skip ci]\n\n## [1.0.1](https://github.com/xmlking/ngx-starter-kit/compare/v1.0.0...v1.0.1) (2018-09-09)\n\n\n### Bug Fixes\n\n* **build:** now updating version ([2d913ea](https://github.com/xmlking/ngx-starter-kit/commit/2d913ea))\n\n\n\n" | npx commitlint`
 
Environment to reproduce:
https://github.com/xmlking/ngx-starter-kit/blob/master/package.json  

## Expected behavior

we should generate release commit message `subject` without numbers. 

## Environment

 
    "semantic-release": "^15.9.14",
    "@semantic-release/changelog": "^3.0.0",
    "@semantic-release/git": "^7.0.3",
    "@semantic-release/github": "^5.0.4",
    "@semantic-release/npm": "^5.0.4",
 
logs:
```
* **build:** now updating version ([2d913ea](https://github.com/xmlking/ngx-starter-kit/commit/2d913ea))

✖   subject must not be sentence-case, start-case, pascal-case, upper-case [subject-case]
✖   found 1 problems, 0 warnings
husky > commit-msg hook failed (add --no-verify to bypass)


    at makeError (/SPA/ngx-starter-kit/node_modules/@semantic-release/git/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/SPA/ngx-starter-kit/node_modules/@semantic-release/git/node_modules/execa/index.js:278:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 1,
  stdout: '',
  stderr:
   'husky > pre-commit (node v10.7.0)\n\u001b[?25lRunning tasks for {apps,libs}/**/*.{ts,json,md,scss} [started]\nRunning tasks for {apps,libs}/**/*.{ts,json,md,scss} [skipped]\n→ No staged files match {apps,libs}/**/*.{ts,json,md,scss}\n\u001b[?25h\u001b[?25hhusky > commit-msg (node v10.7.0)\n⧗   input: \nchore(release): 1.0.1 [skip ci]\n\n* **build:** now updating version ([2d913ea](https://github.com/xmlking/ngx-starter-kit/commit/2d913ea))\n\n✖   subject must not be sentence-case, start-case, pascal-case, upper-case [subject-case]\n✖   found 1 problems, 0 warnings\nhusky > commit-msg hook failed (add --no-verify to bypass)\n',
  failed: true,
  signal: null,
  cmd:
   'git commit -m chore(release): 1.0.1 [skip ci]\n\n## [1.0.1](https://github.com/xmlking/ngx-starter-kit/compare/v1.0.0...v1.0.1) (2018-09-09)\n\n\n### Bug Fixes\n\n* **build:** now updating version ([2d913ea](https://github.com/xmlking/ngx-starter-kit/commit/2d913ea))\n\n\n\n',
  timedOut: false,
  killed: false,
  pluginName: '@semantic-release/git' }npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ngx-starter-kit@0.0.0-semantically-released semantic-release: `semantic-release`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ngx-starter-kit@0.0.0-semantically-released semantic-release script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

**Comment by pvdlg** [comment]
The subject `chore(release): 1.0.1 [skip ci]` should work as it's valid per [the spec](https://conventionalcommits.org/#spec-item-5):
> A description MUST immediately follow the type/scope prefix. The description is a short description of the code changes, e.g., fix: array parsing issue when multiple spaces were contained in string.

The spec also [doesn't enforce any particular casing](https://conventionalcommits.org/#are-the-types-in-the-commit-title-uppercase-or-lowercase):
> Are the types in the commit title uppercase or lowercase?
> Any casing may be used, but it’s best to be consistent.

Anyway, it seems it's a bug in commitlint that fails to detect the case when the subject starts with a version number.
I would suggest to open a bug there.

**Comment by pvdlg** [comment]
Closing this one as it seems it's going to be addressed in marionebl/commitlint#438
